php_laravel/app/Http/Controllers/ChargeController.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+use Illuminate\Http\RedirectResponse;
+use OmiseCharge;
+use OmiseSource;
+use OmiseToken;
+use Illuminate\Support\Facades\Http;
+use App\Models\Payment;
+
+class ChargeController extends Controller
+{
+    //
+    public function index() : View {
+        return view('formcharge');
+    }
+
+    /**
+     * 基本的にchargeをするときは、tokenを作って、それを元にchargeを作るものと理解してる
+     */
+    public function cardCharge(Request $req) : View {
+
+        // print_r($req->name);
+        \Log::debug($req->name);
+
+        $token = OmiseToken::create(array(
+            'card' => array(
+              'name' => $req->name,
+              'number' => trim($req->card_number),
+              'expiration_month' => $req->expired_month,
+              'expiration_year' => $req->expired_year,
+              'security_code' => $req->security_code,
+            )
+          ));
+        
+
+        $charge = OmiseCharge::create(array(
+            'amount' => $req->money,
+            'currency' => 'jpy',
+            'card' => $token['id']
+        ));
+
+        // print_r($charge);
+
+        return view('charge-result', ['charge'=>$charge]);
+    }
+
+    /**
+     * 
+     */
+    public function cardChargeOmise(Request $req) : View {
+
+        \Log::debug($req);
+
+        $charge = OmiseCharge::create(array(
```

Comment:
I am aware `array()` still works but don't you think it's cleaner to use `[]` in place of `array()`?

Reply:
Sure, sorry.
I just forgot basic usage of array.

Reply:
@aashishgurung 
Done
https://github.com/omise/examples/pull/29/commits/64d875dae2316e04d267dd2750d5c4c4f75ceae5